### ISSUE TITLE
Issue with ubercharge

### ISSUE THREAD

**Issue by maximilianrob** [50DKP/FF2-Official#448]
Well, i guess, it is fixes it, added condition (Ubercharged) to healer
```
public Action:Timer_Uber(Handle:timer, any:medigunid)
{
	new medigun=EntRefToEntIndex(medigunid);
	if(medigun && IsValidEntity(medigun) && CheckRoundState()==1)
	{
		new client=GetEntPropEnt(medigun, Prop_Send, "m_hOwnerEntity");
		new Float:charge=GetEntPropFloat(medigun, Prop_Send, "m_flChargeLevel");
		if(IsValidClient(client, false) && IsPlayerAlive(client) && GetEntPropEnt(client, Prop_Send, "m_hActiveWeapon")==medigun)
		{
			new target=GetHealingTarget(client);
			if(charge>0.05)
			{
				TF2_AddCondition(client, TFCond_HalloweenCritCandy, 0.5);
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>TF2_AddCondition(client, TFCond_Ubercharged, 0.5);<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<    
				if(IsValidClient(target, false) && IsPlayerAlive(target))
				{
					TF2_AddCondition(target, TFCond_HalloweenCritCandy, 0.5);
					uberTarget[client]=target;
				}
				else
				{
					uberTarget[client]=-1;
				}
			}
		}

		if(charge<=0.05)
		{
			CreateTimer(3.0, Timer_ResetUberCharge, EntIndexToEntRef(medigun), TIMER_FLAG_NO_MAPCHANGE);
			FF2flags[client]&=~FF2FLAG_UBERREADY;
			return Plugin_Stop;
		}
	}
	else
	{
		return Plugin_Stop;
	}
	return Plugin_Continue;
}
```

**Comment by winstliu** [comment]
@maximilianrob can you remind me what issue you're fixing here?

**Comment by Nopied** [comment]
@50Wliu I think he fixed bug that remove uber invincibility before `m_flChargeLevel` is reached at `0.0`.

**Comment by naydef** [comment]
@50Wliu Another way to fix issue #408
Personally, this should be fixed within FF2. Medigun having longer ubercharge than usual is done by FF2 and the plugin should fix any side effects it does

**Comment by naydef** [comment]
Needs testing, but I assume it's fixed
